components/patient-data/impl/src/test/java/org/phenotips/data/internal/controller/VariantListControllerTest.java

```diff
@@ -88,7 +88,7 @@
 
     private static final String VARIANT_KEY = "cdna";
 
-    private static final String GENESYMBOL_KEY = "genesymbol";
+    private static final String GENESYMBOL_KEY = "gene";
```

Comment:
These are no longer gene symbols, so the name of this constant should be changed.